### PR TITLE
fix: modify code style from older (var, promises) to modern (let/const, async/await)

### PR DESCRIPTION
**Pull request by aleattene**

### CODE REVIEW

**Comment by Gsantomaggio** [comment]
@micheleriva WDYT?

**Comment by aleattene** [comment]
Now everything should work, send_stream and receive_stream.

File: examples/stream_queues/receive_stream.js [hunk 1]
```diff
@@ -1,43 +1,52 @@
...
 
-var amqp = require('amqplib');
+const amqp = require('amqplib');
 
-amqp.connect('amqp://localhost').then(function (conn) {
-    process.once('SIGINT', function () { conn.close(); });
-    return conn.createChannel().then(function (ch) {
 
-        var q = 'my_first_stream';
+async function receiveStream() { //=====================================================================================
```
Comment (micheleriva) [inline examples/stream_queues/receive_stream.js:6]: @aleattene what's the `//=====================================================================================` comment for? Can we get rid of that?

File: examples/stream_queues/receive_stream.js [hunk 2]
```diff
@@ -1,43 +1,52 @@
...
+
+    try {
+        const conn = await amqp.connect('amqp://localhost');
+        process.once('SIGINT', () => { conn.close(); });
```
Comment (micheleriva) [inline examples/stream_queues/receive_stream.js:10]: We can get rid of the curly braces around `conn.close();` here

File: examples/stream_queues/receive_stream.js [hunk 3]
```diff
@@ -1,43 +1,52 @@
...
+
+        const ch = await conn.createChannel();
```
Comment (micheleriva) [inline examples/stream_queues/receive_stream.js:12]: Please use an explicative variable name `ch -> channel`

File: examples/stream_queues/receive_stream.js [hunk 4]
```diff
@@ -1,43 +1,52 @@
...
+        const q = 'my_first_stream';
```
Comment (micheleriva) [inline examples/stream_queues/receive_stream.js:13]: Please use an explicative variable name `q -> queue`

File: examples/stream_queues/receive_stream.js [hunk 5]
```diff
@@ -1,43 +1,52 @@
...
-        });
-    });
-}).catch(console.warn);
\ No newline at end of file
+        console.log(' [*] Waiting for messages. To exit press CTRL+C');
+    }
+    // Catch and display any errors in the console
+    catch(e) { console.log(e) }
+}
+//======================================================================================================================
```
Comment (micheleriva) [inline examples/stream_queues/receive_stream.js:45]: Is that comment necessary?

File: examples/stream_queues/send_stream.js [hunk 6]
```diff
@@ -1,30 +1,44 @@
 #!/usr/bin/env node
 
-var amqp = require('amqplib');
+const amqp = require('amqplib');
 
-amqp.connect('amqp://localhost').then(function (conn) {
-    return conn.createChannel().then(function (ch) {
-        var q = 'my_first_stream';
+
+async function sendStream () { //=======================================================================================
```
Comment (micheleriva) [inline examples/stream_queues/send_stream.js:6]: Is that comment necessary?

File: examples/stream_queues/send_stream.js [hunk 7]
```diff
@@ -1,30 +1,44 @@
...
+
+    try {
+        const conn = await amqp.connect('amqp://localhost');
```
Comment (micheleriva) [inline examples/stream_queues/send_stream.js:9]: Please use an explicative variable name `conn -> connection`

File: examples/stream_queues/send_stream.js [hunk 8]
```diff
@@ -1,30 +1,44 @@
...
+        const ch = await conn.createChannel();
```
Comment (micheleriva) [inline examples/stream_queues/send_stream.js:10]: Please use an explicative variable name `ch -> channel`

File: examples/stream_queues/send_stream.js [hunk 9]
```diff
@@ -1,30 +1,44 @@
...
+        const q = 'my_first_stream';
```
Comment (micheleriva) [inline examples/stream_queues/send_stream.js:11]: Please use an explicative variable name `q -> queue`

File: examples/stream_queues/send_stream.js (hunk 7 above)
Comment (micheleriva) [reply examples/stream_queues/send_stream.js:9]: Also, don't you need the `process.once('SIGINT', conn.close);` control here?

File: examples/stream_queues/receive_stream.js (hunk 2 above)
Comment (micheleriva) [reply examples/stream_queues/receive_stream.js:10]: You can use the shorthand:

`process.once('SIGINT', conn.close);`

**Review by micheleriva — CHANGES_REQUESTED** [review]

File: examples/stream_queues/receive_stream.js (hunk 1 above)
Comment (aleattene) [reply examples/stream_queues/receive_stream.js:6]: Yes we can.

File: examples/stream_queues/receive_stream.js (hunk 5 above)
Comment (aleattene) [reply examples/stream_queues/receive_stream.js:45]: Only for my personal greater readability. 
Unnecessary. I remove it.

File: examples/stream_queues/send_stream.js (hunk 6 above)
Comment (aleattene) [reply examples/stream_queues/send_stream.js:6]: Unnecessary. I remove it.

File: examples/stream_queues/send_stream.js (hunk 7 above)
Comment (aleattene) [reply examples/stream_queues/send_stream.js:9]: Right. Control added. Thanks.

**Comment by Gsantomaggio** [comment]
@aleattene @micheleriva going to merge. Thank you both!

**Pull request merged**